### PR TITLE
Hotfix/use viewport from scene

### PR DESCRIPTION
**Pull request by hingew**
### Description

The viewport from the window needs to be used, otherwiese the alignment dosnt work corretly.

### CODE REVIEW

File: src/SmartSelect/Alignment.elm [hunk 1]
```diff
@@ -1,46 +1,148 @@
...
+                , y = select.element.y + select.element.height
+                , width = select.element.width
+                }
+            )
+            Below
+
+
+getElements : String -> String -> Task Dom.Error Params
+getElements containerId selectId =
+    Task.succeed (\container select viewport -> Params { container = container, select = select, viewport = viewport })
```
Comment (mschindlerMM) [inline src/SmartSelect/Alignment.elm]: Why are we mapping to `Params` here? Couldn't we map to `Alignment` directly?

Then we wouldn't need to expose and handle a `Params` type

File: src/MultiSelect.elm [hunk 2]
```diff
@@ -568,7 +549,8 @@ viewCustom { isDisabled, selected, options, optionLabelFn, optionDescriptionFn,
                             [ ( String.join " "
                                     [ classPrefix ++ "options-container"
                                     , classPrefix ++ "multi-bg-color"
-                                    , Alignment.containerClass classPrefix model.alignment
+
+                                    --, Alignment.containerClass classPrefix model.alignment
```
Comment (mschindlerMM) [inline src/MultiSelect.elm]: Can be removed

File: src/SingleSelect.elm [hunk 3]
```diff
@@ -491,16 +466,9 @@ viewCustom { isDisabled, selected, options, optionLabelFn, optionDescriptionFn,
...
-                        [ id (classPrefix ++ "container")
-                        , class
-                            (String.join " "
-                                [ classPrefix ++ "options-container"
-                                , classPrefix ++ "single-bg-color"
-                                , Alignment.containerClass classPrefix model.alignment
-                                ]
-                            )
-                        ]
+                    Alignment.view
```
Comment (mschindlerMM) [inline src/SingleSelect.elm:469]: This `view` is missing for the MultiSelect

**Review by mschindlerMM — CHANGES_REQUESTED** [review]

File: src/SmartSelect/Alignment.elm (hunk 1 above)
Comment (hingew) [reply src/SmartSelect/Alignment.elm]: Yes, i will update it.

File: src/SmartSelect/Alignment.elm [hunk 4]
```diff
@@ -1,46 +1,148 @@
...
+
+type Placement
     = Above
     | Below
 
 
-init : { container : Element, select : Element } -> Alignment
-init { container, select } =
+type Params
+    = Params { container : Element, select : Element, viewport : Dom.Viewport }
```
Comment (mschindlerMM) [inline src/SmartSelect/Alignment.elm]: We can remove the `Params` type now.

Otherwise LGTM

**Review by mschindlerMM — CHANGES_REQUESTED** [review]

File: src/SmartSelect/Alignment.elm (hunk 4 above)
Comment (hingew) [reply src/SmartSelect/Alignment.elm]: :checkered_flag:

**Review by mschindlerMM — APPROVED** [review]

**Pull request merged**